LEGO1/bumpbouy.cpp

```diff
@@ -0,0 +1,18 @@
+#include "bumpbouy.h"
+
+// 0x100f0394
+static char* g_bumpBouyString = "BumpBouy";

```

Comment:
This is almost certainly not a static string. It's probably an inlined string that's been moved to the data section of the PE, as I explained in an earlier comment.